stat/distuv/gamma.go

```diff
@@ -124,9 +128,8 @@ func (g Gamma) Rand() float64 {
 
 		// Algorithm adjusted to work in log space as much as possible.
 		lambda := 1/a - 1
-		lw := math.Log(a) - 1 - math.Log(1-a)
-		lr := -math.Log(1 + math.Exp(lw))
-		lc, _ := math.Lgamma(a + 1)
+		w := 1 / lambda / math.E
+		lr := -math.Log1p(w)
```

Comment:
Please revert this. The original code intentionally works in log space (as described in the comment above).

Reply:
But it still does!

Reply:
The question that is in my mind is whether `-math.Log1p(1 / lambda / math.E)` is better or worse than `-math.Log1p(math.Exp(-math.Log(lambda) - 1))`. I think you're probably right, that it's better. (They look to be about 1 ULP different in the end for reasonable small alpha.)

Given this, keep yours. Sorry.